### PR TITLE
Nov23

### PR DESCRIPTION
**Pull request by scarletnorberg**
Adding back campaigns that got lost

### CODE REVIEW

**Comment by haozturk** [comment]
@scarletnorberg can you fix the merge conflicts, please?

**Comment by scarletnorberg** [comment]
Sorry I missed this comment. Should be fixed.

**Comment by scarletnorberg** [comment]
Adding in PRCAMPAIGNS-243 PRCAMPAIGNS-244 PRCAMPAIGNS-245 PRCAMPAIGNS-246 PRCAMPAIGNS-247 PRCAMPAIGNS-249

These are the FastSim campaigns and because of the conflict I figured adding this in to this one was better
It is these campaigns:
RunIISpring21UL*FSSIMDR* * is 16,17,18 and the second * is Premix and not Premix

File: campaigns.json [hunk 1]
```diff
@@ -3176,13 +3255,6 @@
        "T2_CH_CERN_HLT"
     ],
     "secondaries": {
-      "/Neutrino_E-10_gun/RunIIFall17FSPrePremix-PUFSUL16_106X_mcRun2_asymptotic_v16-v1/PREMIX" :
```
Comment (haozturk) [inline campaigns.json:3091]: Why do you remove this secondary? @scarletnorberg

Comment (scarletnorberg) [reply campaigns.json:3091]: Because it was not needed

Comment (scarletnorberg) [reply campaigns.json:3091]: The other FS campaign had two although I do not think both are needed. Jordan confirmed that only one is needed on the JIRA ticket.

File: campaigns.json [hunk 2]
```diff
@@ -1413,7 +1413,54 @@
...
+      }, 
+      "/MinBias_TuneCP5_13TeV-pythia8/RunIISummer19UL16SIM-106X_mcRun2_asymptotic_v3-v1/GEN-SIM": {
+       "SiteWhitelist": [
+         "T1_US_FNAL"
+        ]
+      }
+    }, 
+    "secondary_AAA": false, 
+    "tune": true
+  }
```
Comment (haozturk) [inline campaigns.json:1463]: missing coma @scarletnorberg

Comment (scarletnorberg) [reply campaigns.json]: Added

**Pull request merged**